resource_obmcs_database_db_system_test.go

```diff
@@ -77,10 +54,37 @@ func (s *ResourceDatabaseDBSystemTestSuite) TestAccResourceDatabaseDBSystem() {
 	resource.Test(s.T(), resource.TestCase{
 		Providers: s.Providers,
 		Steps: []resource.TestStep{
+			// verify create
 			{
 				ImportState:       true,
 				ImportStateVerify: true,
-				Config:            s.Config,
+				Config: s.Config + `
+				resource "oci_database_db_system" "t" {
+					availability_domain = "${data.oci_identity_availability_domains.ADs.availability_domains.0.name}"
+					compartment_id = "${var.compartment_id}"
+					subnet_id = "${oci_core_subnet.t.id}"
+					database_edition = "ENTERPRISE_EDITION"
+					disk_redundancy = "NORMAL"
+					shape = "VM.Standard1.2"
+					cpu_core_count = "2"
+					ssh_public_keys = ["ssh-rsa KKKLK3NzaC1yc2EAAAADAQABAAABAQC+UC9MFNA55NIVtKPIBCNw7++ACXhD0hx+Zyj25JfHykjz/QU3Q5FAU3DxDbVXyubgXfb/GJnrKRY8O4QDdvnZZRvQFFEOaApThAmCAM5MuFUIHdFvlqP+0W+ZQnmtDhwVe2NCfcmOrMuaPEgOKO3DOW6I/qOOdO691Xe2S9NgT9HhN0ZfFtEODVgvYulgXuCCXsJs+NUqcHAOxxFUmwkbPvYi0P0e2DT8JKeiOOC8VKUEgvVx+GKmqasm+Y6zHFW7vv3g2GstE1aRs3mttHRoC/JPM86PRyIxeWXEMzyG5wHqUu4XZpDbnWNxi6ugxnAGiL3CrIFdCgRNgHz5qS1l MustWin"]
+					display_name = "-tf-db-system"
+					domain = "mycompany.com"
+					hostname = "myOracleDB"
+					data_storage_size_in_gb = "256"
+					license_model = "LICENSE_INCLUDED"
+					node_count = "1"
+					db_home {
+						db_version = "12.1.0.2"
+						display_name = "-tf-db-home"
+						database {
+							"admin_password" = "BEstrO0ng_#11"
+							"db_name" = "aTFdb"
+							character_set = "AL32UTF8"
+							ncharacter_set = "AL16UTF16"
+						}
+					}
+				}`,
 				Check: resource.ComposeTestCheckFunc(
 					resource.TestCheckResourceAttrSet(s.ResourceName, "id"),
```

Comment:
Could definitely use some more validation on this test - but that can be done separately.